java/src/main/bin/build.sh

```diff
@@ -8,7 +8,7 @@ __brew_not_installed=2
 
 make_base="cd /vowpal_wabbit;
 make clean;
-make;"
+make vw java;"
 
 ubuntu_base="apt-get update -qq;
 apt-get install -qq software-properties-common g++ make libboost-all-dev default-jdk;"
```

Comment:
This can be made slightly faster by changing `libboost-all-dev` to `libboost-program-options-dev`.